client/src/components/Card/Card.css

```diff
@@ -81,20 +89,23 @@
   max-height: 202px;
   min-height: 200px;
 }
+
 .Card--mobile img {
   transform: scaleX(0.5);
   margin: 3vh 0 0 0;
 }
+
 .Card--mobile div {
   text-align: center;
-
   padding-bottom: 3vh;
   width: 100%;
 }
+
 .Card--mobile h2 {
   text-align: center;
   width: 100%;
 }
```

Comment:
Can you create classnames that are inline with the naming convention? (i.e. `.Card--mobile__title` instead of `.Card--mobile h2`)